src/services/user.service.js

```diff
@@ -0,0 +1,45 @@
+const { User } = require('../models/User.model.js');
+
+const getUsers = async () => {
+  const users = await User.findAll();
+
+  return users;
+};
+
+const createUser = (name) => {
+  return User.create({ name });
+};
+
+const getUserById = async (id) => {
+  return User.findByPk(id);
+};
+
+const updateUser = async ({ id, name }) => {
+  const [numberOfAffectedRows] = await User.update({ name }, { where: { id } });
```

Comment:
The `update` method returns an array where the first element is the number of affected rows. Ensure that the `name` parameter is valid and properly validated before calling the `update` method to avoid potential issues with no rows being affected.